lib/autoload/kuroko2/workflow/task/fork.rb

```diff
@@ -43,8 +43,9 @@ def extract_child_nodes
         end
 
         def create_child_token(child_node:, env: {})
-          attributes = token.attributes.except('id', 'uuid', 'script', 'path', 'message', 'created_at', 'updated_at')
+          attributes = token.attributes.except('id', 'uuid', 'script', 'path', 'message', 'created_at', 'updated_at', 'context')
           attributes = attributes.merge(uuid: SecureRandom.uuid, parent: token, script: child_node.to_script, path: '/')
+          attributes['context'] = token.context.deep_dup
```

Comment:
(nit) this assignment can be merged into the above line

```ruby
attributes = token.attributes.except('id', 'uuid', 'script', 'path', 'message', 'created_at', 'updated_at', 'context')
attributes = attributes.merge(uuid: SecureRandom.uuid, parent: token, script: child_node.to_script, path: '/', context: token.context.deep_dup)
```

Reply:
Nitpicked. https://github.com/cookpad/kuroko2/pull/128/commits/174918faef34709c042137b3d35e3daae69dc55b